_posts/en/newsletters/2025-04-04-newsletter.md

```diff
@@ -0,0 +1,569 @@
+---
+title: 'Bitcoin Optech Newsletter #348'
+permalink: /en/newsletters/2025/04/04/
+name: 2025-04-04-newsletter
+slug: 2025-04-04-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- secp256k1lab FIXME:harding
+
+## Changing consensus
+
+_A monthly section summarizing proposals and discussion about changing
+Bitcoin's consensus rules._
+
+- **Multiple discussion about quantum computer theft and resistance:**
+  several conversations examined how Bitcoiners could respond to quantum
+  computers becoming powerful enough to allow stealing bitcoins.
+
+  - *Should vulnerable bitcoins be destroyed?* Jameson Lopp [posted][lopp
+    destroy] to the Bitcoin-Dev mailing list several arguments for the
+    destruction of bitcoins vulnerable to quantum theft after an upgrade
+    path to [quantum resistance][topic quantum resistance] has been
+    adopted and users have had time to adopt the solution.  Some
+    arguments include:
+
+    - *Argument from common preference:* he believes most people would
+      prefer that their funds be destroyed rather than stolen by someone
+      with a fast quantum computer.  Especially, he argues, since the
+      thief will be it the set of "the few privileged folks who gain
+      early access to quantum computers".
+
+    - *Argument from common harm:* many of the stolen bitcoins will have
+      been either lost coins or those that were planned to be held
+      long-term.  By contrast, the thieves might quickly spend their
+      stolen bitcoins, resulting in reduced purchasing power for other
+      bitcoins (similar to money supply inflation).  He notes that lower
+      purchasing power of bitcoins reduces miner income, reducing
+      network security, and (in his observation) results in lower
+      merchant acceptance of bitcoins.
+
+    - *Argument from minimal benefit:* although allowing theft could be
+      used to fund the development of quantum computing, stealing coins
+      does not provide any direct benefit to honest participants in the
+      Bitcoin protocol.
+
+    - *Argument from clear deadlines:* nobody can know far in advance
+      the date at which someone with a quantum computer can begin
+      stealing bitcoins, but a specific date at which quantum-vulnerable
+      coins will be destroyed can be announced far in advance with
+      perfect precision.  That clear deadline will provide more
+      incentive for users to upgrade in time, ensuring fewer total coins
```

Comment:
```suggestion
      perfect precision.  That clear deadline will
      incentivize users to move their coins in time, ensuring fewer total coins
```
(to me, "upgrade" sounds like installing new software, but I think what's meant is moving coins to QR-safe UTXOs)